Input/IME/Sdl/SdlIMEHandler.cs

```diff
@@ -0,0 +1,20 @@
+﻿using Microsoft.Xna.Framework;
+
+namespace Rampastring.XNAUI.Input.IME.Sdl
+{
+    /// <summary>
+    /// Integrate IME to DesktopGL(SDL2) platform.
+    /// </summary>
+    internal sealed class SdlIMEHandler(Game game) : IMEHandler
+    {
+        public override bool Enabled { get => false; protected set => _ = value; }
+
+        public override void StartTextComposition()
+        {
+        }
+
+        public override void StopTextComposition()
+        {
+        }
```

Comment:
Should leave a note that this is a stub and the reasons why it's not implemented/an issue link to track the status of the issue ticket that we depend on here.